spherical-gear.h

```diff
@@ -1,34 +1,38 @@
 #include <Servo.h> // for the motor
 #include <cmath> // for the math
 #define TIME_ITERATION 10 // used for servo delay
+#define ANGLE_CONVERT_1
+#define ANGLE_CONVERT_2
 class SphericalGear {
   private:
     Servo servo1; // servo object for motor that changes theta
     Servo servo2; // servo object for motor that changes phi
-    double theta;
-    double phi;
+    double angle1;
+    double angle2;
 	public:
 		SphericalGear(int pin1, int pin2) {
       // initializing the motors
       servo1.attach(pin1);
       servo2.attach(pin2);
       // initializing the angles
-      theta = 0;
-      phi = 0;
+      angle1 = 0;
+      angle2 = 0;
 		}
     void move(double delta_theta, double delta_phi, double angular_speed) {
       // radius is assumed to be 1
+      // convert from spherical angle to motor angle
       // measured in degrees; from mr. nims's equation
-      double total_angle_change = acos( sin(theta)*sin(theta+delta_theta)*cos(phi-delta_phi) + cos(theta)*cos(theta+delta_theta) );
+      double total_angle_change = acos( sin((angle1*ANGLE_CONVERT_1))*sin((angle1*ANGLE_CONVERT_1)+delta_theta)*cos((angle2*ANGLE_CONVERT_2)-delta_phi)
+       + cos((angle1*ANGLE_CONVERT_1))*cos((angle1*ANGLE_CONVERT_1)+delta_theta) );
       double time = total_angle_change / angular_speed; // in miliseconds
       double theta_speed = delta_theta / time;
       double phi_speed = delta_phi / time;
```

Comment:
change theta_speed  and phi_speed to angle1_speed and angle2_speed. And make sure to divide by angle convert